src/SuperDumpService/Program.cs

```diff
@@ -5,7 +5,7 @@ namespace SuperDumpService {
 	public static class Program {
 		public static void Main(string[] args) {
 			var host = new WebHostBuilder()
-				.UseKestrel()
+				.UseKestrel(opt => opt.Limits.MaxRequestBodySize = 8589934592L)  // 8gb
```

Comment:
To be honest, I think the better solution would be to set this limit to long.MaxValue. This would restore the original behavior and prevent that ugly error message from showing up (which now it always does if the upload size limit is exceeded because the kestrel MaxRequestBodySize check is done prior to our the MultipartBodyLengthLimit check)